graphql/types.lua

```diff
@@ -318,7 +318,19 @@ types.long = types.scalar({
 })
 
 local function isFloat(value)
-  return type(value) == 'number'
+  if type(value) == 'number' then
+    return true
+  end
+
+  if type(value) == 'cdata' then
+    if ffi.istype('int64_t', value) then
+      return value >= -2^53 and value < 2^53
+    elseif ffi.istype('uint64_t', value) then
+      return value < 2^53
```

Comment:
Also I guess we need to be sure that comparison between LL/ULL and Lua numbers produces correct result (https://github.com/tarantool/tarantool/issues/4089). Probably tonumber should be additionally applied